parts/k8s/cloud-init/artifacts/cse_customcloud.sh

```diff
@@ -59,16 +59,16 @@ configureK8sCustomCloud() {
         K8S_CLIENT_CERT_PATH="${KUBERNETES_FILE_DIR}/k8s_auth_certificate.pfx"
         echo $SERVICE_PRINCIPAL_CLIENT_SECRET_CERT | base64 --decode > $K8S_CLIENT_CERT_PATH
         # shellcheck disable=SC2002
-        echo $(cat "${AZURE_JSON_PATH}" | \
+        cat "${AZURE_JSON_PATH}" | \
```

Comment:
Just curious how this passes our previously merged "eliminate unnecessary cat" operations PR?

Reply:
Because of the `# shellcheck disable=SC2002`, which makes `shellcheck` give that rule a pass on the subsequent line. 

This was one of two usages (the other is just below in the same file) where we're modifying a file in-place with `jq`, and the other approach of using a temp file seemed uglier.